### ISSUE TITLE
Deprecation of the U.S. survey foot

### ISSUE THREAD

**Issue by dbhart** [ucum-org/ucum#296]
This is an editorial comment regarding the section on the U.S. units of survey length equal to 1200/3937 m. NIST issued an opinion in the Federal Register (U.S. legal regulations) of a final decision to deprecate the survey foot from use in any new or derived works, at least official U.S. works. The international foot is now the official foot to be used in the U.S. This means that there is an opportunity to add "default" foot, mile, etc. to the international customary units, e.g., [ft], [mi]. It may be worth noting this information in the text regarding the U.S. survey units, or simply adding the reference and notice from it.

The following notice is taken from NIST HB-44, 2024 ed., Apdx. C, pp. C-10 and 11 (see footnotes).

NOTICE: In collaboration, National Oceanic and Atmospheric Administration (NOAA) and NIST have taken action to provide national uniformity in the measurement of length. The final decision to retire the U.S. survey foot was published in the Federal Register, announcing the deprecation date of December 31, 2022. Beginning on January 1, 2023, the U.S. survey foot should be avoided, except for historic and legacy applications and will be superseded by the international foot definition (i.e., 1 foot = 0.3048 meter exactly) in all applications. Prior to this date, except for the mile and square mile, the cable’s length, chain, fathom, furlong, league, link, rod, pole, perch, acre, and acre-foot were previously only defined in terms of the U.S. survey foot. With this update, relationships are available in terms of the international foot, which can simply be referred as the “foot.” Either the term “foot” or “international foot” may be used, as required for clarity in technical applications. This is particularly the case for surveying and mapping applications, although over time “foot” will become more prevalent. The preferred measurement unit of length in the United States is the meter (m) and surveyors, map makers, and engineers are encouraged to adopt the SI for their work. For more information see Federal Register (October 5, 2020, 85 FR 62698, p. 62698) available at https://www.govinfo.gov/content/pkg/FR-2020-10-05/pdf/2020-21902.pdf.

**NIST Handbook 44, 2024 ed.** Konijnenburg J, Lee GD, Minnich LB, Williams JS, Warfield L, and Lippa K (2024). **Specifications, Tolerances, and Other Technical Requirements for Weighing and Measuring Devices** as adopted by the *108th National Conference on Weights and Measures*. (Washington, DC: National Institute of Standards and Technology). https://doi.org/10.6028/NIST.HB.44-2024

**Comment by wt** [comment]
I came across this issue when I was reading the spec. The spec says this:

> Customary units are defined in The Unified Code for Units of Measure in order to accommodate practical needs. However metric units are still preferred and the customary symbols should not interfere with metric symbols in any way. Thus, customary units are “stigmatized” by enclosing them into square brackets.

This request appears to be in conflict with that text. I just wanted to point that out in case those reviewing this request find that relevant.

**Comment by chgessner** [comment]
Not sure where you see the conflict. All I see is that this "international foot" definition in current UCUM exactly correponds to [ft_i], numerically. In UCUM the link to SI is constructed via the definition of [in_i] in terms of cm. OTOH, the "survey foot" is numerically equivalent to UCUM's [ft_us]. If I understand the request correctly, it asks for addition of symbols [ft], [in], [mi] as new "aliases" for the existing corresponding symbols [ft_i], [mi_i] etc. and possibly deprecate the latter.

**Comment by wt** [comment]
@chgessner I thought it was asking for the addition of units like "ft" directly, but I think you are correct. Thanks for pointing that out.

**Comment by dbhart** [comment]
Yes, I did intend to note that there is no longer a conflict between the U.S. legal foot and the international foot - the older, "U.S. survey foot" has been legally phased out. However, almost all of the older 'customary' units in the UCUM have modifiers such as '_us' or '_i' or '_av', etc. So, it probably would not be appropriate to put in any plain modifiers for [ft] or [in], etc.

What was done more recently, though, is that the U.S. Government **redefined** the meaning of square-mile, acre and (Gunter's) chain, rod, furlong, and link to be based on the international foot [ft_i] instead of the now-deprecated [ft_us]. Which means that, according to NIST, it would be appropriate to have [smi_i], [acr_i], [ch_i], [rd_i], [fur_i], [lk_i] as units, as they are now the "legally defined" U.S. customary units. However, the reason they redefined the units rather than doing something more complicated is because at the level of precision of Gunter's survey units, the difference between [ft_us] and [ft_i] is too small to matter.

Basically, though this may be of note for an editorial comment in the text, I'm not sure enough people would use a hypothetical [acr_i], for example, to warrant its addition, nor would there be much of a difference in value. 

If any of these were to be added, I would suggest it be only [acr_i], since it is defined by NIST as: 1 acre := 43560 [ft_i] (it's not based on rods or furlongs anymore) and [smi_i], 1 square mile := [mi_i]2, because the square mile is specifically called out by NIST as being based on the international mile, not survey mile, in NIST HB-44.2023 and later editions.

**Comment by chgessner** [comment]
So we would not change the definitions of the now deprecated US units, but instead add the "[X_i]" versions for some (or all??) of the affected units. To be discussed.

**Comment by dbhart** [comment]
I recommend closing this in favor of discussing the specific units that have immediate, real world usage, proposed in #389 and #390 (acre and acre-foot), rather than the units less likely to be used in the real world (rod, chain, etc.).
